### PR TITLE
Add `date_started`, `date_ended`, and `room_name` to the `Meeting` model

### PR DESCRIPTION
**Pull request by nooreldeensalah**
- Added `room_name` and `active_status` fields for the `Meeting model`.
- Implemented an endpoint to get active meetings.
- ~~Refactored POST and PATCH request endpoints.~~
- ~~Added `.title()` method to name fields in models as an attempt to avoid duplicate entries.~~
	- Added non-case sensitive checking to avoid duplicate entries such as `Noor` and `noor` at insertion.
- Implemented the functionality to end a meeting in PATCH requests.
- Fixed potential bugs and added error handling.

### CODE REVIEW

**Comment by AbdulrhmnGhanem** [comment]
I think storing a `started` date and `ended` date is more useful. Then the query will check if the end date is null to return active rooms.

File: functions/dashboard/main.py [hunk 1]
```diff
@@ -166,19 +198,24 @@ def patch_dispatcher(request) -> Union[dict, None]:
...
 
 
-def patch_meeting(resource, request_body):
-    resource.date = request_body.get("date", resource.date)
+def patch_meeting(resource, request_body, request_method):
+    resource.room_name = request_body.get("room_name", resource.room_name)
     resource.link = request_body.get("link", resource.link)
-    resource.participants = request_body.get("participants", resource.participants)
-    resource.labels = request_body.get("labels", resource.labels)
+    # resource.date_started = request_body.get("date_started", resource.date_started) # Requires changes + not sure if needed
```
Comment (AbdulrhmnGhanem) [inline functions/dashboard/main.py:208]: Is this dead?

**Review by AbdulrhmnGhanem — DISMISSED** [review]

File: functions/dashboard/main.py (hunk 1 above)
Comment (nooreldeensalah) [reply functions/dashboard/main.py:208]: It's not exactly dead, but insertion of children has to be done the same way as in the function `add_participants_and_labels_to_meeting` beginning at line 119.

Plus, I don't get know if we will need this functionality? (Modifying the list of meetings for a participant), but we can modify the list of participants for a meeting.

Comment (AbdulrhmnGhanem) [reply functions/dashboard/main.py:208]: No, we won't change the meetings for a participant.

**Review by AbdulrhmnGhanem — APPROVED** [review]

**Pull request merged**